ui.apps/src/main/content/META-INF/vault/filter.xml

```diff
@@ -3,13 +3,7 @@
     <filter root="/apps/cq/core/content/nav/tools/etoolbox/lists"/>
     <filter root="/apps/wcm/core/content/sites/actions/secondary/create/items/createlist"/>
 
-    <filter root="/apps/etoolbox-authoring-kit/clientlib-injector"/>
-    <filter root="/apps/etoolbox-authoring-kit/components"/>
-    <filter root="/apps/etoolbox-authoring-kit/coral-overlay"/>
-    <filter root="/apps/etoolbox-authoring-kit/depends-on"/>
-    <filter root="/apps/etoolbox-authoring-kit/depends-on-injector"/>
-    <filter root="/apps/etoolbox-authoring-kit/install"/>
-    <filter root="/apps/etoolbox-authoring-kit/lists"/>
+    <filter root="/apps/etoolbox-authoring-kit"/>
```

Comment:
Be careful specifying a "too generic" folder because will sweep away everything that is stored under rit, In my current AEM installation, there are folders named "fixes" and "samples" that will just disappear. Needs clarifying